From d3a64c1f3ab387cb06014fff7eec9a6a373f9965 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cristian=20Rodr=C3=ADguez?= <crrodriguez@opensuse.org>
Date: Fri, 20 Feb 2015 15:35:11 -0300
Subject: [PATCH] shared: AFS is also a network filesystem

(cherry picked from commit ba89821c104d959082aad6f3f0e05a8afd575023)
(cherry picked from commit 3c350019fe9e4be16bf110988c324cfa3a21c61b)
---
 src/shared/util.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/util.c b/src/shared/util.c
index 6f22f2b04e..4e8d1a0d5f 100644
--- a/src/shared/util.c
+++ b/src/shared/util.c
@@ -1587,6 +1587,7 @@ bool chars_intersect(const char *a, const char *b) {
 
 bool fstype_is_network(const char *fstype) {
         static const char table[] =
+                "afs\0"
                 "cifs\0"
                 "smbfs\0"
                 "sshfs\0"
